sample-resource-plugin/src/integrationTest/java/org/opensearch/sample/nonsystemindex/ResourceNonSystemIndexSIDisabledTests.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sample.nonsystemindex;
+
+import java.util.Map;
+
+import org.junit.ClassRule;
+
+import org.opensearch.painless.PainlessModulePlugin;
+import org.opensearch.sample.nonsystemindex.plugin.ResourceNonSystemIndexPlugin;
+import org.opensearch.test.framework.cluster.ClusterManager;
+import org.opensearch.test.framework.cluster.LocalCluster;
+
+import static org.opensearch.security.spi.resources.FeatureConfigConstants.OPENSEARCH_RESOURCE_SHARING_ENABLED;
+import static org.opensearch.test.framework.TestSecurityConfig.AuthcDomain.AUTHC_HTTPBASIC_INTERNAL;
+import static org.opensearch.test.framework.TestSecurityConfig.User.USER_ADMIN;
+
+/**
+ * These tests run with resource sharing enabled but the plugin does not declare a system index and system index protection is disabled
+ */
+public class ResourceNonSystemIndexSIDisabledTests extends AbstractResourcePluginNonSystemIndexTests {
```

Comment:
Why make the distinction? I think its reasonable to assume that resource index will always be a system index and will not be accessed directly by regular users. If a plugin declares a resource index without also specifying it as a system index then consider that invalid. If there was a way to hook into the bootup flow then it would be good to have a log message accordingly.

Reply:
removed these tests to avoid confusion.